### PR TITLE
Feature/sp 6 welcome screen

### PR DESCRIPTION
**Pull request by Thongvor**
walkthrough welcome screen with sing up and login buttons.

### CODE REVIEW

File: spendsterIOS/WalkthroughPageViewController.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    func didUpdatePageIndex(currentIndex: Int)
+}
+
+class WalkthroughPageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    weak var walkthroughDelegate: WalkthroughPageViewControllerDelegate?
+    let pageSubheadings = ["Welcome to Budget Planner", "Save money with ease", ""]
+    let pageText = ["Take control of your money\nand save them by tracking your expenses", "Easily create budgets, and see our\nsuggestions based on your spending", "Take control of your money\nand save them by tracking your expenses."]
+    let pageImages = ["Illustration", "hand-pork", "illustration-3"]
+    var currentIndex = 0
+    override func viewDidLoad() {
```
Comment (bogdanustyak) [inline spendsterIOS/WalkthroughPageViewController.swift]: @Thongvor please add empty line

**Review by bogdanustyak — CHANGES_REQUESTED** [review]

File: spendsterIOS/WalkthroughPageViewController.swift (hunk 1 above)
Comment (Thongvor) [reply spendsterIOS/WalkthroughPageViewController.swift]: the empty line above each line of code here?

Comment (bogdanustyak) [reply spendsterIOS/WalkthroughPageViewController.swift]: @Thongvor functions should be separated by empty lines, not fields

**Review by bogdanustyak — CHANGES_REQUESTED** [review]

File: spendsterIOS/WalkthroughPageViewController.swift (hunk 1 above)
Comment (Thongvor) [reply spendsterIOS/WalkthroughPageViewController.swift]: @bogdanustyak Got it

File: spendsterIOS/CornerButton.swift [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 3/27/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+// swiftlint:disable force_cast
+import Foundation
+import UIKit
+
+@IBDesignable class CornerButton: UIButton {
```
Comment (bogdanustyak) [inline spendsterIOS/CornerButton.swift]: @Thongvor please create a "Presentation" group and move all UI files from "spensterIOS" into this sub group.

**Review by bogdanustyak — CHANGES_REQUESTED** [review]

**Comment by bogdanustyak** [comment]
@Thongvor please do the same with welcome screens classes inside Presentation group

**Review by bogdanustyak — APPROVED** [review]

**Pull request merged**